tests/sentry/auth/test_access.py

```diff
@@ -19,16 +19,24 @@
 from sentry.testutils.silo import control_silo_test
 
 
+class AccessFactoryTestCase(TestCase):
+    def setUp(self) -> None:
+        self.factory = access.OrmAccessFactory()
```

Comment:
Ideally, we'd like to finagle this so that it runs every test case twice, once with each concrete `AccessFactory` type. Maybe a light dusting of metaclass magic.

Reply:
Probably yes, unfortunately.  testunit doesn't give us parameterization like pytest does, so we'll have to duplicate methods in the end.